.github/workflows/release.yaml

```diff
@@ -38,14 +38,14 @@ jobs:
         run: rm -rf node_modules/@changesets/*/node_modules/prettier
 
       - name: Create Release Pull Request or Publish to npm
-        working-directory: lit
         # The id of this step must not be "changesets", or else the step will be invisible
         # in the list of steps from the GitHub UI when the action runs (though it will still
         # run, and its output will appear in the raw logs). Unknown why this is the case,
         # see https://github.com/changesets/action/issues/149 for discussion.
         id: cs
         uses: changesets/action@v1
         with:
+          cwd: lit
```

Comment:
Is it worth adding a comment about `working-directory` only working with `run` and not `uses`?

Reply:
sgtm, added.